src/cli.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _CRETRO_SRC_CLI_H_
+#define _CRETRO_SRC_CLI_H_
+
+#include <stdint.h>
+
+typedef struct arg_conf {
+    int debug;
```

Comment:
What does `debug` mean here? Isn't this `log_lvl`?

Reply:
`debug` originally was a `bool`, I haven't changed the name since I introduced multiple debug levels. Can change it